### PR TITLE
Document Compact blob store task

### PR DESCRIPTION
**Pull request by mosabua**
https://issues.sonatype.org/browse/NEXUS-9653

Rendered output at http://bamboo.s/browse/BOOK-NEXUS78

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -329,6 +329,9 @@ task.
 
 The following task types are available to perform specific maintenance:
 
+Compact blob store:: Deletions of binary assets in a blob store are only removing the reference. This does not
```
Comment (mrprescott) [inline chapter-configuration.asciidoc]: Content deleted from a blob store isn't deleted physically from disk, but are 'soft-deleted' (internally marked for deletion). This task performs the actual deletion of the relevant files, and frees up the storage space.

Comment (mosabua) [inline chapter-configuration.asciidoc]: Haha... jumping the gun there @mrprescott .. I just did this to show @dulanism the process... I will fix it all up now ;-)

Comment (mosabua) [inline chapter-configuration.asciidoc]: But this is a good example how a review can help a lot!

Comment (dulanism) [inline chapter-configuration.asciidoc]: No knowledge wasted @mosabua. Thx for the update @mrprescott

**Comment by kellyrob99** [comment]
+1

**Pull request merged**